core/domain/prod_validation_jobs_one_off.py

```diff
@@ -43,6 +43,8 @@
     models.NAMES.topic, models.NAMES.user
 ])
 
+FULLY_VALIDATED_CONSTANT = 'fully-validated'
```

Comment:
VALIDATION_STATUS_SUCCESS or VALIDATION_STATUS_FULLY_VALIDATED

("constant" is vague, the names above follow the pattern for constants `<item type>_<item name>`)

Reply:
Done.